paper-dropdown.js

```diff
@@ -85,7 +86,7 @@ import PaperDropdownBehavior from './paper-dropdown-element-behavior';
  * @element paper-dropdown
  * @demo demo/index.html
  */
-class PaperDropDown extends Polymer.mixinBehaviors([PaperDropdownBehavior], Polymer.Element) {
```

Comment:
Mixin is a way in polymer to abstract out code / functionality which can be overridden by someone using the component.
Example: the validation in `IronValidatableBehavior` which we override for our custom validation. Which in turn can be overridden by someone using `paper-dropdown`. Also, removing the `IronValidatableBehavior` should result in form validation to fail. Did you test it?